packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -49,6 +49,8 @@ const FormLabel = ({
     );
   }
 
+  const isLabelLeftPositioned = position === 'left' && isDesktop;
```

Comment:
Fixes a bug where if you make the window smaller the label will go to the top but the size of the label will still remain like the desktop version. 

<img width="203" alt="image" src="https://user-images.githubusercontent.com/35374649/189107025-5e62549f-a7a5-4aa3-b97c-d6767b2bf42e.png">